sycl/test-e2e/Basic/intel-ext-device.cpp

```diff
@@ -1,10 +1,7 @@
 // RUN: %clangxx -fsycl %s -o %t.out
-// RUN: env ONEAPI_DEVICE_SELECTOR=level_zero:gpu %t.out
-// RUN: env ONEAPI_DEVICE_SELECTOR=opencl:gpu %t.out
+// REQUIRES: level_zero || opencl
+// RUN: %GPU_RUN_PLACEHOLDER %t.out
 //
-// REQUIRES: gpu
-// Temporarily disable on L0 due to fails in CI
-
 //==--------- intel-ext-device.cpp - SYCL device test ------------==//
 //
 // Returns the low-level device details.  These are Intel-specific extensions
```

Comment:
Let's fix the comment here.

// Returns the low-level device details.  These are Intel-specific extensions
// that are only supported on Level Zero **or OpenCL**.